AST.fs

```diff
@@ -153,6 +168,22 @@ module Pretty =
     open Starling.Core.Model.Pretty
     open Starling.Core.Var.Pretty
 
+    /// Some ANSI color codes for nice outputting during debugging
+    /// wraps x as "<[line n: x]>" with color formats
+    let pprint_lines = false
```

Comment:
I wonder if there's a portable library for colouring console output for F#, that might be cleaner?